src/providers/response-types/kaltura-flavor-asset-list-response.ts

```diff
@@ -18,19 +18,16 @@ export class KalturaFlavorAssetListResponse extends BaseServiceResult {
   private setDataWithoutDuplicates(flavors: Array<any>): Array<KalturaFlavorAsset> {
     const data = [];
     // sort flavors by height and filter out flavors with height 0
-    const sortedFlavors = flavors
-      .sort((a: {height: number}, b: {height: number}) => {
-        return a.height < b.height ? 1 : -1;
-      })
-      .filter(flavor => {
-        return flavor.height > 0;
-      });
+    const sortedFlavors = flavors.sort((a: {height: number}, b: {height: number}) => {
+      return a.height < b.height ? 1 : -1;
```

Comment:
```suggestion
      return a.height - b.height;
```

Reply:
you mean b - a in that case

Reply:
you're right
so b.height - a.height it is

Reply:
done